model/src/w3wavemd.F90

```diff
@@ -2450,15 +2449,6 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
           END IF
 #endif
 #endif
-
-!
-! This barrier is from older code versions. It has been removed in 3.11
-! to optimize IO2/3 settings. May be needed on some systems still
-!
-!!/MPI              IF (FLAG0) CALL MPI_BARRIER (MPI_COMM_WCMP,IERR_MPI)
```

Comment:
Just wanted to draw attention to this line of code, just in case anyone has used this since 3.11 and there's any need to keep this around. Tagging a few others for awareness/thoughts: @ukmo-ccbunney @mickaelaccensi

Reply:
I don't use it